utils-metrics/pom.xml

```diff
@@ -120,6 +120,7 @@
     <dependency>
       <groupId>org.bdgenomics.utils</groupId>
       <artifactId>utils-misc_2.10</artifactId>
+      <scope>compile</scope>
```

Comment:
compile is the default scope; for some reason IDEA was giving me trouble unless I made it explicit.  Probably just user error, I'm an emacs guy